### PR TITLE
Quick fixes & Maintenance

### PR DESCRIPTION
**Pull request by kinisoftware**
* Using new logo
 * Adding Google Analytics
 * Updating deprecated/broken links
 * Fixing the repo/contributors listing

### CODE REVIEW

File: css/materialize.min.css
Comment (kinisoftware) [inline css/materialize.min.css:1]: I know, usually this file must not be modified but we are not autogenerating it anymore so it should be fine.

File: css/style.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
 }
 
 .tabs .indicator {
-	background-color: #EE9941;
+	background-color: #fe5000;
```
Comment (kinisoftware) [inline css/style.css:13]: New "orange" color

File: index.html [hunk 2]
```diff
@@ -1,9 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <!-- Google tag (gtag.js) -->
```
Comment (kinisoftware) [inline index.html:4]: Adding Google Analytics

File: js/init.js [hunk 3]
```diff
@@ -27,8 +27,7 @@
       });
 	},
 		populateRepositoryTab = function(repoName, tabId) {
-			$.get("//api.github.com/users/"+ repoName + "/repos", 
-				{"access_token" : "c1ed8f00d37bcd5b91fb80b07dc149b10f5f7e35"}, // NOTE: this token is not valid. No need to report this for our bug bounty program.
+			$.get("//api.github.com/users/"+ repoName + "/repos",
```
Comment (kinisoftware) [inline js/init.js:30]: The API request works without the token but the rate limit is lower. More info: https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limiting

**Review by hdost — APPROVED** [review]

**Pull request merged**